### ISSUE TITLE
List of languages is empty

### ISSUE THREAD

**Issue by tomitrescak** [gadicc/meteor-messageformat#128]
Hi, I tried to search for the solution between issues, but can't seem to find it.
My problem is that the list of languages is empty and I have to manually navigate to transitions via URL. I followed the guide closely but no luck. See screenshot. Thanks!

![screen shot 2015-05-19 at 13 02 34](https://cloud.githubusercontent.com/assets/2682705/7694730/d2a6b448-fe27-11e4-874d-b6940297eb55.png)

Also with semanticui the admin looks a bit odd.

![screen shot 2015-05-19 at 13 07 17](https://cloud.githubusercontent.com/assets/2682705/7694737/015509f2-fe28-11e4-80f4-d29eca2029f1.png)

**Comment by gadicc** [comment]
Hey, I'm happy you figured out you can still get to the other languages by hand :>  That actually tells me a lot.  This is all on the stable v0 release, right?  It's not a requirement, but can you see if you init with `sendPolicy: 'all'` if it makes a difference?

**Comment by gadicc** [comment]
Re the theme, it's fixed on the v2 branch.

**Comment by tomitrescak** [comment]
Are you sure? I am using v2 now, and looks pretty much the same. Using following versions:

```
msgfmt:core                    2.0.0-preview.11  MessageFormat support, the Meteor way
msgfmt:extract                 2.0.0-preview.8  Extracts native / translatable strings from your app
msgfmt:ui                      2.0.0-preview.2  messageformat: translation UI
```

**Comment by gadicc** [comment]
Oh yeah, sorry, I should have mentioned it's [fixed](http://messageformat-v2.meteor.com/msgfmt/locale/all/0) but not published yet.  It will be in msgfmt:ui@2.0.0-preview.3.

I'll have to think what could cause this situation (with no languages) in v2.  I've only seen it before when it said there were 0 native strings, and we just fixed that.  Anything particular about your setup?  appcache, browserpolicy, etc?

**Comment by tomitrescak** [comment]
Not really, I have actually added it to the clean application.

**Comment by tomitrescak** [comment]
On other thing I noticed. When I change english translation in the interface, it gets overwritten back at next server reload. Not sure if that's a bug or a feature.

**Comment by gadicc** [comment]
No, setLocale() should persist, either by localStorage or from Meteor user record if the user is logged in.  You're calling `mf.init('en');` in your common code, right?  Can you maybe also paste the full log from the console?  All the debug messaging is enabled by default in the pre-releases.

**Comment by tomitrescak** [comment]
There is not much in the console. 

```
=> App running at: http://localhost:3000/
I20150520-10:37:06.036(10)? 2015-05-20 10:37:06.034 info:  [msgfmt:extracts]  4 string(s) added, 0 changed, and 0 marked as removed.
=> Client modified -- refreshing
I20150520-10:40:49.951(10)? NODE_ENV=development
=> Meteor server restarted
I20150520-10:41:39.898(10)? NODE_ENV=development
=> Meteor server restarted
```

This is my config file
```javascript
//file:/lib/config.js

msgfmt.init('en');

After second restart the translations disappear and are reset to original text stored in templates.
Just to make sure you understand the scenario:

1. I lunch two browsers (Safari and Firefox)
1. In Safari I log in and modify the English translation
1. The translation appears in Safari after I manually reload the page 2 times!
1. In Firefox, where I am logged out, the translation is also visible
1. I make a change to the code and a hot-code reload happens
1. Translation are reset to original

**Comment by tomitrescak** [comment]
There is one more thing that came to my mind. My deployment is a bit special, since I am using Typescript and I am using gulp to build my project. The gulp file removes ALL the javascript files in the project directory before every build and then replaces them with the newly built ones. I notices that when I only change something in the generated js files, the translations survive the hot code reload, but when I rebuild the application they disappear.

You can download the application from here (there are also really simple instructions on how to make it run):

https://github.com/tomitrescak/meteor-boilerplate-typescript

I want to make your package as a core localisation package.
